packages/app-project/src/shared/components/SubjectSetPicker/SubjectSetPicker.js

```diff
@@ -88,18 +88,20 @@ function SubjectSetPicker ({
             const href = `${baseUrl}/workflow/${workflow.id}/subject-set/${subjectSet.id}`
             const panoptesCompleteness = subjectSet.completeness[workflow.id]
             return (
-              <Link
-                key={subjectSet.id}
-                href={addQueryParams(href)}
-                passHref
-              >
-                <Anchor>
-                  <SubjectSetCard
-                    {...subjectSet}
-                    completeness={panoptesCompleteness}  /* This will override subjectSet.completeness */
-                  />
-                </Anchor>
-              </Link>
+              <Box data-testid={`test-subject-set-link-${subjectSet.id}`} key={subjectSet.id}>
```

Comment:
Why are these wrapped in a flexbox now?

Reply:
There's no way to add a `className` or a `data-testid` to the `Link` that wraps the SubjectSetCard component. Open to other suggestions so that I can grab that component without too much fuss / assumption about DOM-level implementation.

Reply:
`Link` is an abstract component. The HTML link is rendered by the Grommet `Anchor`. Grommet components will pass DOM attributes through to the HTML element so you should be able to modify the link anchor.

Reply:
Gotcha - moved to `<Anchor>` element

Reply:
Here's the docs for `next/link` as a wrapper for links:
https://nextjs.org/docs/pages/api-reference/components/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag